contributing/versions.md

```diff
@@ -13,23 +13,27 @@ MDC's version numbers strictly follow [semantic versioning](http://semver.org/):
 
 Major releases can contain breaking changes to clients, while minor and patch releases cannot.
 
-## What are breaking changes?
+## How do we determine which numbers need to change?
 
-- API deletions or modifications.
+We follow the guidelines defined by the [semver v2.0.0](https://semver.org/spec/v2.0.0.html#semantic-versioning-specification-semver) specification.
+
+## Roughly-speaking, what are breaking changes?
+
+- Build-breaking modifications for either Objective-C or Swift code.
 - Visible changes to user interface components.
```

Comment:
Should this remain limited to  *visible changes* or should we expand this to include any user-facing change including non-visible changes?  For example, changing a Chip view to use CustomActions instead of surface subviews is currently a non-breaking change (it is not visible) but probably should be due to the nature of the UX.

Reply:
In your example does that work count as a refactoring toward expected behavior, or would a client reasonably consider that to be a breaking behavioral change?